src/Microsoft.Extensions.Configuration.UserSecrets/build/netstandard1.0/Microsoft.Extensions.Configuration.UserSecrets.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(UserSecretsAssemblyAttributeFile)' == '' and $(MSBuildProjectFile.EndsWith('.vbproj')">
```

Comment:
Turns out there is not only a `Language` property, but an actual MSBuild task dedicated to this kind of think. "WriteCodeFragment" https://msdn.microsoft.com/en-us/library/ff598685.aspx